metrics/platform-metrics-ui/src/main/webapp/js/view/Metrics.view.js

```diff
@@ -0,0 +1,111 @@
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define([
+        'backbone',
+        'jquery',
+        'underscore',
+        'marionette',
+        'handlebars',
+        'icanhaz',
+        'text!templates/metrics.handlebars'
+    ],
+    function (Backbone, $, _, Marionette, Handlebars, ich, metricsTemplate) {
+
+        var MetricsView = {};
+        ich.addTemplate('metricsTemplate', metricsTemplate);
+
+        Handlebars.registerHelper('titleizeCamelCase', function (camelCase) {
+            // insert a space before all caps
+            return camelCase.replace(/([A-Z])/g, ' $1')
+                // uppercase the first character
+                .replace(/^./, function (str) {
+                    return str.toUpperCase();
+                });
+        });
+
+        MetricsView.MetricsPage = Marionette.Layout.extend({
+            template: 'metricsTemplate',
+            initialize: function () {
+                _.bindAll(this);
+            },
+            events: {
+                "click #download-summary": "downloadSummary"
+            },
+            downloadSummary: function (e) {
+                e.preventDefault();
+                var range = this.$('#range')[0].value;
+                var unit = this.$('#unit')[0].value;
+                var interval = this.$('#interval')[0].value;
+                var endDate = new Date();
+                switch (unit) {
+                    case 'year':
+                        endDate.setUTCMonth(0);
+                        endDate.setUTCDate(0);
+                        endDate.setUTCHours(0);
+                        endDate.setUTCMinutes(0);
+                        break;
+                    case 'month':
+                        endDate.setUTCMonth(endDate.getUTCMonth(), 0);
+                        endDate.setUTCHours(0);
+                        endDate.setUTCMinutes(0);
+                        break;
+                    case 'week':
+                        endDate.setUTCDate(endDate.getUTCDate() - endDate.getUTCDay());
+                        endDate.setUTCHours(0);
+                        endDate.setUTCMinutes(0);
+                        break;
+                    case 'day':
+                        endDate.setUTCHours(0);
+                        endDate.setUTCMinutes(0);
+                        break;
+                    case 'hour':
+                        endDate.setUTCMinutes(0);
+                        break;
+                }
+                endDate.setUTCSeconds(0);
+                endDate.setUTCMilliseconds(0);
+                var startDate = new Date(endDate);
+                while (range > 0) {
```

Comment:
After talking with @kguerriero, this math doesn't look right to me. I'll fix it.